src/elements/ProjectJobs/projectJobs.utils.js

```diff
@@ -15,6 +16,12 @@ export const getJobsStatistics = (jobs, match, scheduledJobs, workflows) => {
       0
     )
   }
+  if (workflows.data) {
```

Comment:
Since we don't only want to know `workflows.data` is defined, but we are going to use an array method on it, it might be better to make sure it is an array:
```suggestion
  if (Array.isArray(workflows.data)) {
```